tests/testthat/test-filter_exist.R

```diff
@@ -0,0 +1,65 @@
+# filter_exist ----
+## Test 1: filter_exist() works as expected ----
+test_that("filter_exist Test 1: filter_exist() works as expected", {
+  input_dataset <- tibble::tribble(
+    ~USUBJID, ~AGE, ~SEX,
+    "01-701-1015", 63, "F",
+    "01-701-1034", 77, "F",
+    "01-701-1115", 84, "M",
+    "01-701-1444", 63, "M"
+  )
+
+  input_dataset_add <- tibble::tribble(
+    ~USUBJID, ~AEDECOD, ~AESTDTC,
+    "01-701-1015", "DIARRHOEA", "2014-01-09",
+    "01-701-1034", "APPLICATION SITE PRURITUS", "2014-08-27",
+    "01-701-1034", "FATIGUE", "2014-11-02",
+    "01-701-1115", "FATIGUE", "2013-01-14"
+  )
```

Comment:
```suggestion
  input_dataset <- tibble::tribble(
    ~USUBJID,     ~AGE, ~SEX,
    "01-701-1015",  63,  "F",
    "01-701-1034",  77,  "F",
    "01-701-1115",  84,  "M",
    "01-701-1444",  63,  "M"
  )

  input_dataset_add <- tibble::tribble(
    ~USUBJID,                         ~AEDECOD,     ~AESTDTC,
    "01-701-1015",                 "DIARRHOEA", "2014-01-09",
    "01-701-1034", "APPLICATION SITE PRURITUS", "2014-08-27",
    "01-701-1034",                   "FATIGUE", "2014-11-02",
    "01-701-1115",                   "FATIGUE", "2013-01-14"
  )
```

Getting styler to comply gets tricky with these, have to left-align the first column, then right-align the rest, mostly aesthetic if you want to fix while the CI/CD stuff is down

Reply:
> Getting styler to comply gets tricky with these, have to left-align the first column, then right-align the rest, mostly aesthetic if you want to fix while the CI/CD stuff is down

Thanks for the tip @zdz2101, I didn't realise this! It's odd because when I run `styler::style_active_file()`, it styles it as the code currently is (i.e. with no alignment). I've manually updated the testthat file, but left the roxygen headers the same (I assume that's okay for the styler checks).